config/src/config/state_sync_config.rs

```diff
@@ -451,7 +458,7 @@ impl Default for AptosDataClientConfig {
             max_epoch_chunk_size: MAX_EPOCH_CHUNK_SIZE,
             max_num_output_reductions: 0,
             max_optimistic_fetch_lag_secs: 20, // 20 seconds
-            max_response_bytes: MAX_MESSAGE_SIZE as u64,
+            max_response_bytes: CLIENT_MAX_MESSAGE_SIZE_V2 as u64,
```

Comment:
We have v1 and v2 for max_network_chunk_bytes.
We have only v2 for max_response_bytes.
What's the logic behind it?